releaselinks.go

```diff
@@ -0,0 +1,167 @@
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/releases/%s/assets/links/%d", url.QueryEscape(project), tag, linkID)
+
+	req, err := s.client.NewRequest("GET", u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var rl ReleaseLink
+	resp, err := s.client.Do(req, &rl)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return &rl, resp, err
+}
+
+// CreateReleaseLinkOptions represents CreateReleaseLink() options.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html#create-a-link
+type CreateReleaseLinkOptions struct {
+	Name string `url:"name" json:"name"`
+	URL  string `url:"url" json:"url"`
+}
+
+// CreateReleaseLink creates a link.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html#create-a-link
+func (s *ReleaseLinksService) CreateReleaseLink(pid interface{}, tag string, opt *CreateReleaseLinkOptions, options ...OptionFunc) (*ReleaseLink, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/releases/%s/assets/links", url.QueryEscape(project), tag)
+
+	req, err := s.client.NewRequest("POST", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var rl ReleaseLink
+	resp, err := s.client.Do(req, &rl)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return &rl, resp, err
+}
+
+// UpdateReleaseLinkOptions represents UpdateReleaseLink() options.
+//
+// You have to specify at least one of Name of URL.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html#update-a-link
+type UpdateReleaseLinkOptions struct {
+	Name string `url:"name,omitempty" json:"name,omitempty"`
+	URL  string `url:"url,omitempty" json:"url,omitempty"`
+}
+
+// UpdateReleaseLink updates an asset link.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html#update-a-link
+func (s *ReleaseLinksService) UpdateReleaseLink(pid interface{}, tag string, linkID int, opt *UpdateReleaseLinkOptions, options ...OptionFunc) (*ReleaseLink, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/releases/%s/assets/links/%d", url.QueryEscape(project), tag, linkID)
+
+	req, err := s.client.NewRequest("PUT", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var rl ReleaseLink
+	resp, err := s.client.Do(req, &rl)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return &rl, resp, err
+}
+
+// DeleteReleaseLink deletes a link from release.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html#delete-a-link
+func (s *ReleaseLinksService) DeleteReleaseLink(pid interface{}, tag string, linkID int, options ...OptionFunc) (*ReleaseLink, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/releases/%s/assets/links/%d", url.QueryEscape(project), tag, linkID)
```

Comment:
For lines of code that are longer than 100 chars, I prefer to use multiple lines:

```suggestion
	u := fmt.Sprintf("projects/%s/releases/%s/assets/links/%d",
		url.QueryEscape(project),
		tag,
		linkID,
	)
```